public/app/core/components/form_dropdown/form_dropdown.ts

```diff
@@ -165,7 +176,7 @@ export class FormDropdownCtrl {
   updateValue(text) {
     text = _.unescape(text);
 
-    if (text === '' || this.text === text) {
+    if ((!this.allowCustom && text === '') || this.text === text) {
```

Comment:
This change removed the feature that make it possible to click the input and get alternatives but then click away and it would keep last value. What was the reason for this change